deployment/terraform/assets/cluster.tf

```diff
@@ -17,6 +17,14 @@ data "aws_region" "current" {}
 
 data "aws_caller_identity" "current" {}
 
+data "http" "my_public_ip" {
+  url = "https://ifconfig.me/ip"
```

Comment:
Could we use https://checkip.amazonaws.com/ as a slightly more "official" service?

Reply:
Oh interesting, wasn't aware of that. Will check

Reply:
I checked, and while AWS refers to that site to check for public IP, I couldn't find any official documentation for the tool. But given it's maintained by AWS, we can use that.